README.md

```diff
@@ -1,3 +1,3 @@
 # notaufnahme-i2b2
-Paketierung der I2B2 Installation für das AKTIN Datawarehouse in Notaufnahmen
 
+Paketierung der I2B2 Installation für das AKTIN Datawarehouse in Notaufnahmen
```

Comment:
Allgemein ist empfohlen, Dateien mit einem newline zu beenden.
https://thoughtbot.com/blog/no-newline-at-end-of-file

Diese Verschiebung dürfte am Markdown nichts ändern.